src/main.rs

```diff
@@ -227,19 +227,34 @@ struct Args {
     enable_logs: Option<bool>,
 
     /// Overrides the OpenTelemetry endpoint used for emitting traces, metrics and logs. This can also be set with `OTEL_EXPORTER_OTLP_ENDPOINT`.
-    #[clap(long = "override-observability-endpoint")]
+    #[clap(
+        long = "override-observability-endpoint",
+        env = "WASMCLOUD_OVERRIDE_OBSERVABILITY_ENDPOINT",
+    )]
     observability_endpoint: Option<String>,
 
     /// Overrides the OpenTelemetry endpoint used for emitting traces. This can also be set with `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`.
-    #[clap(long = "override-traces-endpoint", hide = true)]
+    #[clap(
+        long = "override-traces-endpoint",
+        env = "WASMCLOUD_OVERRIDE_TRACES_ENDPOINT",
+        hide = true
+    )]
     traces_endpoint: Option<String>,
 
     /// Overrides the OpenTelemetry endpoint used for emitting metrics. This can also be set with `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT`.
-    #[clap(long = "override-metrics-endpoint", hide = true)]
+    #[clap(
+        long = "override-metrics-endpoint",
+        env = "WASMCLOUD_OVERRIDE_METRICS_ENDPOINT",
+        hide = true,
+    )]
```

Comment:
Just a note I'm going to remove these since we decided back in the day to override the metrics/traces/logs endpoints using the OTEL specific environment variables instead

Reply:
Hey so I saw the comment above these lines, and tried those other ENV vars and they didn't work well -- I was still seeing it trying to send traces the wrong place. :man_shrugging: